bin/oref0-bluetoothup.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+
+if ! ( hciconfig -a | grep -q "PSCAN" ) ; then
+   sudo killall bluetoothd
+   sudo /usr/local/bin/bluetoothd --experimental &
+fi
+
+if ( hciconfig -a | grep -q "DOWN" ) ; then
+   sudo hciconfig hci0 up
+fi
+
+if !( hciconfig -a | grep -q $HOSTNAME ) ; then
```

Comment:
the name is actually changing the Alias name for the BlueZ, but in order for the phone or device to see that name you must do the power off and power on in the bluetoothctl. so it only helps with pairing, the phone does not really care what the name is, it only looks at the mac address after pairing. So your paired device name only helps you, not the phone.